test/e2e/README.md

```diff
@@ -0,0 +1,80 @@
+# e2e test for Kvrocks Operator
+
+## Cluster setup
+
+These tests musts be running on a Kubernetes cluster, and we provide two ways to create a cluster for end-to-end(e2e) testing:
+
+### Script
+You can use the shell script in `script` to install the cluster.
+
+```bash
+make e2e-setup # create a cluster with minikube 
+```
+
+> Note: The script will install the cluster with minikube by default, if you want to use other tools, you can modify the script. (Welcome to contribute your script to support more tools).
+
+
+Now you can use the following config to test the operator:
+
+```go
+var config = &Config{
+	InstallKubernetes:        false,
```

Comment:
`InstallKubernetes` might not be clear enough, it could easily confuse users. What we're doing here is creating a k8s cluster named 'e2e-test' within minikube. I would also suggest replacing 'InstallKubernetes' in other places where it's used.

Reply:
@ColinChamber Hi，
I have removed this value and now determine whether to install Kubernetes based on whether the `kubeconfig` parameter is empty or not.